volttron/platform/control.py

```diff
@@ -1735,7 +1735,14 @@ def show_serverkey(opts):
 
     return 0 if success, 1 if false
     """
-    q = Query(opts.connection.server.core)
+    conn = opts.connection
+    if not conn:
+        _stderr.write(
+            "VOLTTRON is not running. This command "
+            "requires VOLTTRON platform to be running\n"
+        )
+        return
```

Comment:
as an error this should return 1 or not None or 0

Reply:
Fixed.